pkg/authorization/api/register.go

```diff
@@ -5,7 +5,7 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-const GroupName = ""
+const GroupName = "authorization.openshift.io"
```

Comment:
@deads2k so we need the kube PR to land first on order to be able to merge this?